src/lru_time_cache.rs

```diff
@@ -83,7 +92,13 @@ impl<K: Clone + Eq + Hash, V> LruTimeCache<K, V> {
     /// Removes a key-value pair from the cache, returning the value at the key if the key
     /// was previously in the map.
     pub fn remove(&mut self, key: &K) -> Option<V> {
-        self.map.remove(key).map(|v| v.0)
+        let v = self.map.remove(key).map(|v| v.0);
+        if let Some(ref mut tx) = self.tx {
+            if let Err(e) = tx.try_send(key.clone()) {
+                warn!("failed to notify of remove, {}", e);
```

Comment:
```suggestion
                warn!("Failed to send notification of key removal, {}", e);
```
``notify``, ``notif``, ``notification`` are used throughout this PR in warnings to stay consistent and have clear error handling we should just use notification. Also the warning is a little unclear I put a suggestion which is more inline with what other parts of the code use for channel callback errors

Reply:
My bad I have never read the word notify before 😅